pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -372,7 +373,11 @@ func (l *LibvirtDomainManager) MigrateVMI(vmi *v1.VirtualMachineInstance, option
 }
 
 var updateHostsFile = func(entry string) (err error) {
-	file, err := kutil.OpenFileWithNosec("/etc/hosts", os.O_WRONLY|os.O_APPEND)
+	u, _ := user.Current()
+	if u.Uid != "0" {
+		return nil
+	}
```

Comment:
lets drop this function if we can get away with it now.

If we can't get away with dropping this, I'd rather move the logic regarding whether or not we edit the hosts file up to the function that calls `updateHostsFile` instead. That way we don't silently ignore performing logic when the user isn't root